packages/server/src/automations/automationUtils.ts

```diff
@@ -99,6 +99,12 @@ export function getError(err: any) {
   return typeof err !== "string" ? err.toString() : err
 }
 
+export function validateAttachment(attachmentObject: any) {
```

Comment:
Naming code conventions generally expect validate to return a response. This seems more like `guardAttachment`